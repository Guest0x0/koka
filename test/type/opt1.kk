noinline fun plus(x:int,y:int,z:int=1)
  x+y+z

pub fun main()
  [1,2,3].foldl(0,plus)
