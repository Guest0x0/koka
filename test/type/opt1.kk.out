type/opt1/main: () -> int
type/opt1/plus: (x : int, y : int, z : ? int) -> int